docs/concepts/views.md

```diff
@@ -1,37 +1,19 @@
 # Concept: Views
```

Comment:
Maybe structured vs unstructured views use-cases and scenarios section shoould go here